packages/mui-material/src/StepLabel/StepLabel.js

```diff
@@ -104,7 +104,6 @@ const StepLabelIconContainer = styled('span', {
   slot: 'IconContainer',
   overridesResolver: (props, styles) => styles.iconContainer,
 })({
-  flexShrink: 0, // Fix IE11 issue
```

Comment:
Introduced in https://github.com/mui/material-ui/pull/13375

Tested in Chrome, Firefox, and Safari. Screenshot in Chrome:

<img width="500" alt="Screenshot 2024-05-23 at 16 47 21" src="https://github.com/mui/material-ui/assets/16889233/37972b3d-2b7f-4a33-8382-415f9912bd7f">